enhancements/machine-config/mco-cgroupsv2-support.md

```diff
@@ -201,19 +200,31 @@ The following jobs will be run against cgroup v2 periodically and with a minimum
 
 ### Upgrade / Downgrade Strategy
 
-Downgrading a cluster to an OpenShift version not containing cgroup v2 support
+- Downgrading a cluster to an OpenShift version not containing cgroup v2 support
 is unsupported.
+- Upgrading a cluster which is on cgroup v1 to a version >= 4.19 is blocked until it is migrated to cgroup v2
 
 ### Version Skew Strategy
 
 A cluster installed with cgroup v2 will abide by the usual skew upgrade path.
 
 #### Removing a deprecated feature
 
-N/A
+cgroup v1 support would be removed from the future/associated versions of RHEL and hence the setting of `CgroupMode_V1` has to be removed from OCP clusters(>= 4.19)
```

Comment:
Can you expand on what this means? The CgroupMode_V1 API variable would stick around, since it is in the API. An admission webhook might be needed to deny a user setting CgroupMode_V1 instead.

Reply:
Makes sense